static/js/main.js

```diff
@@ -0,0 +1,129 @@
+
+function preloadArchi()
+{
+    let darkmode = getCookie('darkmode');
+    if(darkmode && darkmode.length > 0)
+    {
+       switch(darkmode)
+       {
+           case "on":
+                switchDarrMode('dark');
+            break;
+            case "off":
+                switchDarrMode('main');
+            break;
+
+       }
+    }
+    else
+    {
+        let mode = getDarkModeInit();
+        switch(mode)
+        {
+            case "dark":
+                    setCookie('darkmode', 'off');
+                break;
+                case "main":
+                case "auto":
+                    setCookie('darkmode', 'on');
+                break;
+
+        }
+    }
+}
+
+function toggleDarkMain()
+{
+    let darkmode = getCookie('darkmode');
+    if(darkmode && darkmode.length > 0)
+    {
+       let mode = getDarkModeInit();
+
+       switch(mode)
+       {
+           case "dark":
+                setCookie('darkmode', 'off');
+                switchDarrMode('main');
+            break;
+            case "main":
+            case "auto":
+                setCookie('darkmode', 'on');
+                switchDarrMode('dark');
+            break;
+
+       }
+    }
+    else
+    {
+        //console.log('no existe');
+        setCookie('darkmode', 'on');
+    }
+}
+
+function getDarkModeInit()
+{
+   let mode = ""; 
+   let likns =  Array.from(document.querySelectorAll('head > link'));
```

Comment:
Can be replaced with document.querySelector("link[media*='prefers-color-scheme']");

Reply:
Done 💯